BUILD.gn

```diff
@@ -247,6 +247,26 @@ if (!is_ios) {
   }
 }
 
+if (!is_android && !is_ios) {
```

Comment:
I think we should just replace `!is_android && !is_ios` by `is_win`.

Reply:
I might remove this - thanks @goodov for the feedback about avoiding convenience aliases. You had mentioned that too @mherrmann. I'll get that fixed up and I think we can remove this section